ios/ReactTestApp/ReactTestApp.common.xcconfig

```diff
@@ -16,3 +16,6 @@ MTL_FAST_MATH = YES
 SDKROOT = iphoneos
 SWIFT_TREAT_WARNINGS_AS_ERRORS = YES
 WARNING_CFLAGS = -Wall
+OTHER_CFLAGS = $(inherited) -fstack-protector-strong -fPIE -Wformat -Wformat-security -Werror=format-security
+OTHER_LDFLAGS = $(inherited) -fstack-protector-strong -fPIE -pie -Wformat -Wformat-security -Werror=format-security
```

Comment:
`-W*` shouldn't be necessary since we have `WARNING_CFLAGS = -Wall` and `GCC_TREAT_WARNINGS_AS_ERRORS = YES`:
```suggestion
OTHER_CFLAGS = $(inherited) -fstack-protector-strong -fPIE
OTHER_LDFLAGS = $(inherited) -fstack-protector-strong -fPIE -pie
```

Also, can you sort the list? 😄

Reply:
Yeah, -Wall enables -Wmost which enables -Wformat which enables -Wformat-securtity. (Most of which are enabled by default anyway...) So yeah, turns out to be redundant. :/